### PR TITLE
Red Hat OpenShift Service on AWS

### PR DESCRIPTION
**Pull request by vkareh**
This renames every instance of MOA to ROSA. This should be both
backwards compatible with the current deployment OCM (using MOA) and
future-proof for a future deployment of OCM (using ROSA).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,19 +26,19 @@ If you get stuck or you are starting out and want more details, the rest of this
 * [Creating admin users for your cluster](#optional-create-dedicated-and-cluster-admins)
 * [Cleaning up](#next-steps)
 
-By the end of this guide you will have an Amazon Red Hat OpenShift cluster running in your AWS account.
+By the end of this guide you will have an Red Hat OpenShift Service on AWS cluster running in your AWS account.
```
Comment (sfortner-RH) [inline README.md:29]: By the end of this guide you will have **a** Red Hat OpenShift Service on AWS cluster running in your AWS account.

File: README.md [hunk 2]
```diff
@@ -282,50 +282,50 @@ Global Flags:
...
-$ moactl create cluster --cluster-name=rh-moa-test-cluster
+$ rosa create cluster --cluster-name=rh-rosa-test
 
-I: Creating cluster with identifier '1de87g7c30g75qechgh7l5b2bha6r04e' and name 'rh-moa-test-cluster'
-I: To view list of clusters and their status, run `moactl list clusters`
-I: Cluster 'rh-moa-test-cluster' has been created.
-I: Once the cluster is 'Ready' you will need to add an Identity Provider and define the list of cluster administrators. See `moactl create idp --help` and `moactl create user --help` for more information.
-I: To determine when your cluster is Ready, run `moactl describe cluster rh-moa-test-cluster`.
+I: Creating cluster with identifier '1de87g7c30g75qechgh7l5b2bha6r04e' and name 'rh-rosa-test'
+I: To view list of clusters and their status, run `rosa list clusters`
```
Comment (sfortner-RH) [inline README.md:302]: To view **a** list of clusters and their status, run `rosa list clusters`

**Review by sfortner-RH — CHANGES_REQUESTED** [review]
Couple very minor edits on my end, otherwise LGTM!

File: README.md (hunk 1 above)
Comment (vkareh) [reply README.md]: Good catch, thanks! Should be fixed now

File: README.md (hunk 2 above)
Comment (vkareh) [reply README.md]: Fixed!

**Review by sfortner-RH — APPROVED** [review]

**Review by lamek — COMMENTED** [review]
LGTM - we need a PR to update the readme.

**Comment by lamek** [comment]
This updates the readme. https://github.com/openshift/moactl/pull/169

File: templates/cloudformation/iam_user_osdCcsAdmin.json [hunk 3]
```diff
@@ -6,8 +6,7 @@
         "ManagedPolicyArns": [
           "arn:aws:iam::aws:policy/AdministratorAccess"
         ],
-        "UserName": "osdCcsAdmin",
-        "Tags" : [{"Key": "MOA", "Value": "true"}]
```
Comment (jharrington22) [inline templates/cloudformation/iam_user_osdCcsAdmin.json:10]: Did you intend to remove these?

Comment (vkareh) [reply templates/cloudformation/iam_user_osdCcsAdmin.json:10]: I don't think we ever used these. I can bring them back if they are useful for something else, or we could rename them to `ROSA`, but I don't find references to it in our code

Comment (jharrington22) [reply templates/cloudformation/iam_user_osdCcsAdmin.json:10]: I remembered them being talked about somewhere it was [here](https://github.com/openshift/moactl/pull/151#discussion_r507683779) - Looks like they are not being used.

Comment (sulmer-redhat) [reply templates/cloudformation/iam_user_osdCcsAdmin.json:10]: It is not working as of right now; I have an open issue in this regard on aws-go-sdk.
When I will make this work, I will open a new PR.

**Pull request merged**